src/workspacer.Bar/Widgets/BatteryWidget.cs

```diff
@@ -28,23 +32,26 @@ public override IBarWidgetPart[] GetParts()
                 {
                     return Parts(Part(currentBatteryCharge.ToString("#0%"), LowChargeColor));
                 }
+                else if (currentBatteryCharge <= MedChargeThreshold)
+                {
+                    return Parts(Part(currentBatteryCharge.ToString("#0%"), MedChargeColor));
+                }
+                else if (currentBatteryCharge <= HighChargeThreshold)
+                {
+                    return Parts(Part(currentBatteryCharge.ToString("#0%"), HighChargeColor));
+                }
```

Comment:
This entire `else if` block can be removed, and `HighChargeColor` should be applied to the final `else` block below.